src/test/java/com/google/api/generator/gapic/composer/ComposerTest.java

```diff
@@ -42,8 +37,13 @@ public class ComposerTest {
   private final List<GapicClass> clazzes =
       Arrays.asList(
           GrpcServiceCallableFactoryClassComposer.instance().generate(context, echoProtoService));
-  private final String protoPackage = context.gapicMetadata().getProtoPackage();
-  private final List<Sample> samples = clazzes.get(0).samples();
+  private final Sample sample =
+      Sample.builder()
+          .setRegionTag(
+              RegionTag.builder().setServiceName("servicename").setRpcName("rpcname").build())
+          .build();
+  private List<Sample> ListofSamples = Arrays.asList(new Sample[] {sample});
```

Comment:
this non-constant class field should use lowerCamelCase

Reply:
thanks! just updated.